controllers/v1/programs.js

```diff
@@ -737,4 +737,46 @@ module.exports = class Programs extends Abstract {
 			}
 		})
 	}
+
+	/**
+     * @api {get} /project/v1/programs/removeSolutionsFromProgramComponent/:_id 
+     * @apiVersion 1.0.0
+     * @apiName removeSolutionsFromProgramComponent
+     * @apiGroup Programs
+     * @apiHeader {String} internal-access-token
+     * @apiSampleRequest /project/v1/programs/removeSolutionsFromProgramComponent/6825a30dc407a50014b6a73f
+	 * Controller method to remove a solution ID from all program documents where it exists.
+	 * Delegates the actual logic to `programsHelper.removeSolutionsFromProgramComponent()`.
+	 *
+	 * @param {Object} req - Express request object containing `params._id` (solutionId).
+	 * @returns {Promise<Object>} - Result of the operation from the helper function.
+     * 
+     * * @apiUse successBody
+     * @apiUse errorBody
+     * @apiParamExample {json} Response:
+    {
+        "message": "Solution and associated resources deleted successfully",
+        "status": 200,
+        "result": {
+            "noSolutionDeleted": 2,
+            "solutionId": "682c0340ba875600144d8ec5",
+            "success": true
+     }
+    }
+	*/
```

Comment:
@MallanagoudaB just call it removeSolutions

Reply:
we already know its program controller etc, no need to add program word everywhere in all function names